src/components/molecules/Downloads/Downloads.md

```diff
@@ -6,10 +6,23 @@ import themes from "../../../theme";
 import Title from "../../atoms/Typography/Title";
 import ThemeTester from "../../../styleguide/ThemeTester";
 import ImageModal from "../../../styleguide/ImageModal";
+import Download from "../../atoms/Download/Download";
 import img1 from "./Downloads.png";
 
 <React.Fragment>
-  <ThemeTester>downloads</ThemeTester>
+  <ThemeTester flexDirection={"column"}>
+    <Downloads
+      title={"Downloads"}
+      subtitle={"Subtitle"}
+      downloads={[
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+        <Download href="https://escolalms.github.io/headless-format/paper.pdf" />,
+      ]}
```

Comment:
what i thought originaly is that we pass `array` of **Data** (DTO) and this component will generate  map of ` <Download />` as a result ...

Reply:
ok, so the structure should be like this?

```
downloads={[
      {
          fileName: "File name",
          href: "https://www.google.com",
      },
      {
          fileName: "File name",
          href: "https://www.google.com",
      }
  ]}
```

Reply:
yep